airbyte-integrations/connectors/source-recurly/metadata.yaml

```diff
@@ -2,26 +2,28 @@ data:
   ab_internal:
     ql: 200
     sl: 100
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.2.0@sha256:c22a9d97464b69d6ef01898edf3f8612dc11614f05a84984451dde195f337db9
   connectorSubtype: api
   connectorType: source
   definitionId: cd42861b-01fc-4658-a8ab-5d11d0510f01
-  dockerImageTag: 0.4.1
+  dockerImageTag: 0.5.0
   dockerRepository: airbyte/source-recurly
   documentationUrl: https://docs.airbyte.com/integrations/sources/recurly
   githubIssueLabel: source-recurly
   icon: recurly.svg
   license: MIT
   name: Recurly
+  registries:
+    cloud:
+      enabled: true
+    oss:
+      enabled: true
+  releaseStage: alpha
```

Comment:
Non-blocking: Can you remind me of the enum of possible release stages? With such a big upgrade, should we consider Recurly to be better than alpha? ;)

Reply:
This is a good question 🤔 The releaseStages are [alpha, beta, GA], but unless I'm mistaken we actually don't care about those values anymore since we moved away from the that terminology in favor of Community/Certified support levels.